x-pack/test/functional/apps/dashboard/group3/reporting/screenshots.ts

```diff
@@ -221,20 +199,6 @@ export default function ({
         await unloadEcommerce();
       });
 
-      it('downloads a PDF file: small dashboard', async function () {
```

Comment:
Removed this test because it is redundant with a test under `Sample data from Kibana 7.6`